frontend/pages/DetailedViewPage.js

```diff
@@ -40,24 +50,45 @@ export default class DetailedViewPage extends Component {
                 </View>
```

Comment:
(commenting on the line closest to the desired line)

changePrice seems to be unused, if it isn't used then it should be removed until it is needed.

Reply:
Yes, unused being added in another PE (#118) Not removing because will be addressed when both PRs are merged into master.